README.md

```diff
@@ -46,6 +46,7 @@ Available Addons:
 - `docker`
 - `phrase_app`
 - `semaphore`
+- [`devise`](https://github.com/heartcombo/devise)
```

Comment:
Let keep the format as normal text without the link same like another addon, for now, @noor-ahmed as the user know what's devise before using it I guess.